graphwebhook/src/main/java/com/example/graphwebhook/TokenHelper.java

```diff
@@ -36,10 +36,6 @@ private TokenHelper() {
     public static boolean isValidationTokenValid(@NonNull final String[] validAudiences,
             @NonNull final String[] validTenantIds, @NonNull final String serializedToken,
             @NonNull final String keyDiscoveryUrl) {
-
-        Objects.requireNonNull(validAudiences);
```

Comment:
any specific reason why we're removing defensive programing here?

Reply:
Basically it's moved further up the stack :D. Because of the `@NonNull` decorator, the analyzer complains about it earlier. The `requireNonNull` is now done on the parameter passed to `areValidationTokensValid`:

```java
areTokensValid = TokenHelper.areValidationTokensValid(new String[] {clientId},
        new String[] {tenantId},
        Objects.requireNonNull(notifications.validationTokens),
        Objects.requireNonNull(keyDiscoveryUrl));
```

Reply:
thanks for clarifying!